pkg/report/linux.go

```diff
@@ -2313,15 +2264,14 @@ var linuxOopses = append([]*oops{
 			},
 		},
 		[]*regexp.Regexp{},
-		crash.UnknownType,
 	},
 	{
 		[]byte("rust_kernel: panicked"),
 		[]oopsFormat{
 			{
 				title:  compile("rust_kernel: panicked"),
 				report: compile("rust_kernel: panicked at [^\n]*?\n(.+?)\n"),
-				fmt:    "%[1]v in %[2]v",
+				fmt:    "rust_kernel panicked: %[1]v in %[2]v",
```

Comment:
We don't yet have existing bugs with old titles, right? Generally it's not OK to rename titles (w/o messing with alt titles, or something else).

Reply:
@nogikh landed it on friday. I don't think we have these bugs.

Reply:
Will it generate "rust: KASAN: ..."? It would look ... too 'rusty'...

Reply:
What does actually 'rust_kernel: panicked' mean?

Reply:
Let me remove the rust part. Too many questions as for the side change.